### PR TITLE
[Selenium] Update selenium tests from dashboard package according to workspace details changes, fix unstable tests

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR:
update selenium tests from ```dashboard``` package according to workspace details changes:
- **CreateAndDeleteProjectsTest** - update project deleting from Dashboard after removing project details page;
- **WorkspaceDetailsMachinesRamTest** - check that Save changes button is disabled if saved RAM value wasn't changed.

fix unstable(flacky) selenium tests:
- **ConfigureClasspathBaseTest** - check that project items is selected in **Project Explorer** before starting configuring classpath;
- **CommandsEditorTest** - check that a test project is selected in **Project Explorer** before starting command that use ```${current.project.path}``` macros;
- **PreviewUrlIntoCommandsEditorTest** - delete this test from CheSuite.xml test suite because its usecases already are checked in other tests(CommandsEditorTest and MacrosCommandsEditorTest tests);
- **AutocompleteFeaturesInEditorTest** - don't check all content in java class, just added;
- **CheckOpenFileFeatureTest** - add test to FLAKY test group.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/12605

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java [hunk 1]
```diff
@@ -125,35 +122,27 @@ public void createProjectTest() {
...
     workspaces.selectWorkspaceItemName(WORKSPACE);
     workspaceDetails.selectTabInWorkspaceMenu(PROJECTS);
-    workspaceProjects.waitProjectIsPresent(WEB_JAVA_SPRING);
-    workspaceProjects.waitProjectIsPresent(CONSOLE_JAVA_SIMPLE);
-    openProjectSettings(WEB_JAVA_SPRING);
-    workspaceProjects.clickOnDeleteProject();
-    workspaceProjects.clickOnDeleteItInDialogWindow();
-    workspaceProjects.waitProjectIsNotPresent(WEB_JAVA_SPRING);
+
+    deleteProject(WEB_JAVA_SPRING);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java:132]: shouldn't we wait on `WEB_JAVA_SPRING` project is not present after deletion as well? `workspaceProjects.waitProjectIsNotPresent(WEB_JAVA_SPRING);`

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java [hunk 2]
```diff
@@ -125,35 +122,27 @@ public void createProjectTest() {
...
     workspaceProjects.waitProjectIsPresent(SECOND_WEB_JAVA_SPRING_PROJECT_NAME);
-    openProjectSettings(CONSOLE_JAVA_SIMPLE);
-    workspaceProjects.clickOnDeleteProject();
-    workspaceProjects.clickOnDeleteItInDialogWindow();
-    workspaceProjects.waitProjectIsNotPresent(CONSOLE_JAVA_SIMPLE);
-  }
 
-  private void switchToWindow(String windowHandle) {
-    seleniumWebDriver.switchTo().window(windowHandle);
+    deleteProject(CONSOLE_JAVA_SIMPLE);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java:136]: shouldn't we wait on `CONSOLE_JAVA_SIMPLE` project is not present after deletion as well? `workspaceProjects.waitProjectIsNotPresent(CONSOLE_JAVA_SIMPLE);`

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java (hunk 1 above)
Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java:132]: Project deleting is checked at the end of ```deleteProject()``` method.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java (hunk 2 above)
Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java:136]: Project deleting is checked at the end of ```deleteProject()``` method.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java (hunk 1 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java:132]: I see.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java (hunk 2 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java:136]: Got it.

**Comment by SkorikSergey** [comment]
ci-build

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CheckOpenFileFeatureTest.java [hunk 3]
```diff
@@ -73,7 +74,7 @@ public void tearDown() throws Exception {
     factoryServiceClient.deleteFactory(FACTORY_NAME);
   }
 
-  @Test
+  @Test(groups = FLAKY)
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CheckOpenFileFeatureTest.java]: There is only one test in the test class. So, we can mark test class itself as FLAKY to exclude whole class from test execution.

**Review by dmytro-ndp — APPROVED** [review]
looks good

**Pull request merged**